fele-client-service/cli/test/cli.spec.js

```diff
@@ -68,6 +85,44 @@ describe('user program test cases', () => {
     expect(createChannel).toHaveBeenCalledTimes(1)
 
   })
+  
+ 
+ 
+  test('COMMANDER CMD TEST for creating a chaincode', async () => {
+    //console.log({...network })
+    const commandArr = [
+        'chaincode',
+        'register',
+        'create',
+        '-nn',
+        'artemis',
+        '-cn',
+        'nasa',
+        'ccn',
+        'assests'
+      ]
+    interpreter.parse(commandArr, { from : 'user'});
+    expect(createChaincode).toHaveBeenCalledTimes(1)
+
+  })
+
+  test('COMMANDER CMD TEST for creating a chaincode', async () => {
+    //console.log({...network })
+   const commandArr = [
+        'chaincode',
+        'register',
+        'create',
+        '--networkName',
+        'artemis',
+        '--channelName',
+        'nasa',
+        '--chaincodeName',
+        'assests'
+      ]
+    interpreter.parse(commandArr, { from : 'user'});
+    expect(createChaincode).toHaveBeenCalledTimes(2)
```

Comment:
Any reason for calling the createChaincode 2 times?

Reply:
It is the second time we are calling the createChaincode command.
1. chaincode register create -nn artemis -cn nasa -ccn assets  
2. chaincode register create --networkName artemis --channelName nasa --chaincodeName assets

Just to check if both these commands are getting called

Reply:
You have to reset the mocks before each test. 

Try adding `beforeEach(() => {
    jest.resetAllMocks();
  });` before the tests